### PR TITLE
RSDK-3613 update motion tutorial code to not use a singularity

### PR DESCRIPTION
**Pull request by raybjork**
I tested this new position works on a UR arm.

### CODE REVIEW

**Review by JessamyT — COMMENTED** [review]
This is fine but this code is carried over from https://github.com/viamrobotics/docs/blob/main/docs/tutorials/services/accessing-and-moving-robot-arm.md, and there's some accompanying body text. Also the code comments here and there should be updated. Mind adding that to this PR?

Also also, this will induce a bigger motion right? Since it's both a lower joint and a bigger angle change? Should we not be worried that people might smack themselves/their surroundings with such a move?

**Comment by raybjork** [comment]
> This is fine but this code is carried over from https://github.com/viamrobotics/docs/blob/main/docs/tutorials/services/accessing-and-moving-robot-arm.md, and there's some accompanying body text. Also the code comments here and there should be updated. Mind adding that to this PR?
> 
> Also also, this will induce a bigger motion right? Since it's both a lower joint and a bigger angle change? Should we not be worried that people might smack themselves/their surroundings with such a move?

I'll make the fixes to the other doc.  I'm not worried about your other question though.  We run through this tutorial weekly and never have had concerns like that.  This movement is followed up by an 100 mm move up in tool-space so there should be no danger posed by this change.

**Comment by raybjork** [comment]
I also want to make a note that I tested this new position and that both it and the subsequent position in the tutorial are reachable from either the xarm or the UR arm.  The motion team QA tests using this tutorial on both arms, so this is an improvement for us

File: docs/tutorials/services/accessing-and-moving-robot-arm.md
Comment (JessamyT) [inline docs/tutorials/services/accessing-and-moving-robot-arm.md:1]: Line 255 references this change so it should be changed to something like:

If you execute the sample joint move statement above, the third joint of your arm should move a small amount (30 degrees).

**Review by JessamyT — APPROVED** [review]
Line 255 still needs changing. Otherwise looks good; thanks for updating.

**Pull request merged**